### ISSUE TITLE
clear session cookies

### ISSUE THREAD

**Issue by curlyfrie** [franmontiel/PersistentCookieJar#6]
Hi,

thx again for the library :)

I added a clearSession method in PersistentCookieJar.. maybe also something useful for the lib.

```
public synchronized void clearSession() {
	this.cache.clear();
	this.cache.addAll(this.persistor.loadAll());
}
```

**Comment by franmontiel** [comment]
Thanks for your contribution. It can be useful in some use cases. Planned for future releases

**Comment by franmontiel** [comment]
Implemented in 1.0.0. Thanks for your contribution @curlyfrie
